buildSrc/src/main/kotlin/care/data4life/gradle/fhir/script/download-scripts.gradle.kts

```diff
@@ -58,7 +58,7 @@ private val repository = "https://raw.githubusercontent.com/d4l-data4life/hc-gra
 private val branch = "main"
 val baseLink = "$repository/$branch"
 val scriptPathSource = "buildSrc/src/main/kotlin/care/data4life/sdk/script"
-val scriptPathTarget = "buildSrc/src/main/kotlin/care/data4life/hl7/fhir"
+val scriptPathTarget = "buildSrc/src/main/kotlin/care/data4life/gradle/datadonation/script"
```

Comment:
should not be `datadonation` -> `fhir`

Reply:
Glad you spotted it!!!!

Reply:
In the line before - It must be also gradle from now on, right?

Reply:
👍